src/app/api/services/guards/auth-guard.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
```

Comment:
Found a bug:

If you are in like: ```/c/calendar``` and you try to go like: ```/c/calendar/asd/```

Reply:
fixed, route for unknown routes (**) was checking if menu was valid. Now just redirect to the main portlet.